tests/app-sync/run_validators.go

```diff
@@ -18,30 +18,23 @@ import (
 	"github.com/testground/sdk-go/runtime"
 )
 
-// This test-case is a 101 on how Celestia-App only should be started.
-// In this test-case, we are testing the following scenario:
-// 1. Every instance can create an account
-// 2. The orchestrator(described more below) funds the account at genesis
-//    and sends the initial genesis.json to the rest of the validators' set
-// 3. After receiving the initial genesis.json, validators are signing the
-//    genesis transaction(gentx)
-// 4. Validators collects all genesis transactions
-// 5. The chain is started
-func initVal(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
+func RunValidator(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*4)
 	defer cancel()
 
-	client := initCtx.SyncClient
-	netclient := network.NewClient(client, runenv)
+	syncclient := initCtx.SyncClient
+	netclient := network.NewClient(syncclient, runenv)
 
 	netclient.MustWaitNetworkInitialized(ctx)
 
 	config := network.Config{
 		Network: "default",
 		Enable:  true,
 		Default: network.LinkShape{
-			Latency:   100 * time.Millisecond,
-			Bandwidth: 1 << 20, // 1Mib
+			// Latency: 100 * time.Millisecond,
+			// Bandwidth: 4 << 26, // 256Mib
+			Bandwidth: 5 << 26, // 320Mib
+			// Bandwidth: 4 << 27, // 512Mib
```

Comment:
the same as above. do we need commented code?

Reply:
No. It was a left-over artifact of the upcoming test-case